content/css/concepts/grids/terms/grid-template-areas/grid-template-areas.md

```diff
@@ -0,0 +1,121 @@
+---
+Title: 'grid-template-areas'
+Description: 'Defines a grid template by referencing the names of the grid areas.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Grid'
+  - 'Layout'
+  - 'Responsive'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `grid-template-areas` property defines a CSS grid by assigning a [`grid-area`](https://www.codecademy.com/resources/docs/css/grids/grid-area) name to each element in the template. The `grid-template` property can be used as a shorthand to define the areas as well as defining the sizes of the columns and rows.
+This property is collectively used with [`grid-template-columns`](https://www.codecademy.com/resources/docs/css/grids/grid-template-columns) and [`grid-template-rows`](https://www.codecademy.com/resources/docs/css/grids/grid-template-rows) under the [`grid-template`](https://www.codecademy.com/resources/docs/css/grids/grid-template) shorthand property.
+
+## Syntax
+
+```css
+.item-one {
+  grid-area: nav;
+}
+.item-two {
+  grid-area: main;
+}
+.item-three {
+  grid-area: footer;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-areas:
+    'nav nav nav'
+    'main main main'
+    'footer footer footer';
+}
+```
+
+Each grid item can be assigned a name with the `grid-area` property. Then, the `grid-template-areas` utilizes these areas in the grid container. For this properties to work, `display: grid` must be set in the `.grid-container`.
+
+![Example](https://raw.githubusercontent.com/Codecademy/docs/main/media/grid-template-area1.png)
```

Comment:
```suggestion
![CSS grid-template-areas Example](https://raw.githubusercontent.com/Codecademy/docs/main/media/grid-template-area1.png)
```

Can we rename this image stored in `media/`?

~`grid-template-area1.png`~ --> `grid-template-area.png`